### ISSUE TITLE
Does not conform to standard nodejs syntax

### ISSUE THREAD

**Issue by WORMSS** [goliatone/arpscan#4]
normally the callback is last after any arguments (including optional arguments).

So I was expecting 

``` javascript
arpscanner(callback); //or
arpscanner(options, callback);
```

Sadly because your syntax is opposite to the rest of nodejs

``` javascript
arpscanner(callback, options);
```

things such as promisify cannot get a handle on your code.

I can fix it my side with a wrapper functions, but just thought I would let you know.
- Colin.

**Comment by nagromc** [comment]
@goliatone may be interested in a PR?

**Comment by WORMSS** [comment]
arpscan-promise.js
````javascript
const arpscan = require("arpscan");

module.exports = function (options) {
    return new Promise((resolve, reject) => {
        arpscan((err, out) => err ? reject(err) : resolve(out))
    }, options);
};
````

**Comment by WORMSS** [comment]
Please see pull request #6 for fix for this.

**Comment by goliatone** [comment]
merged PR #6
